.stylelintrc.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+module.exports = {
+  "ignoreFiles": [
+    "Dockerfile",
+    "LICENSE",
+    "*.*",
+    "images/*.*",
+    "!*.css"
+  ],
+  rules: {
+    "at-rule-no-unknown": true,
```

Comment:
Question: why not just use the standard config of rules in https://github.com/stylelint/stylelint-config-standard?  How did you arrive at this set of rules?

Reply:
I used https://github.com/stylelint/stylelint-config-recommended , should I use the standard config instead of recommended?

Reply:
Instead of manually defining these rules, you can install the recommended:

```
npm install stylelint-config-recommended --save-dev
```

and then use

```
  "extends": "stylelint-config-recommended"
```

And you don't/won't have to specify all these.  You'd only use these `rules` if you wanted to override something, which we might.  This is very similar to [what we're doing in eslint](https://github.com/Seneca-CDOT/telescope/blob/master/.eslintrc.js#L2) with the [Airbnb ruleset](https://www.npmjs.com/package/eslint-config-airbnb)